webapp/locales/de.json

```diff
@@ -515,7 +515,11 @@
     "search": {
         "placeholder": "Suchen",
         "hint": "Wonach suchst Du?",
-        "failed": "Nichts gefunden"
+        "failed": "Nichts gefunden",
+        "heading": {
+          "Post": "Beiträge",
+          "User": "Benutzer"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T11:49:31Z" title="Wednesday, January 8th 2020, 12:49:31 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -515,7 +515,12 @@
     "search": {
         "placeholder": "Suchen",
         "hint": "Wonach suchst Du?",
-        "failed": "Nichts gefunden"
+        "failed": "Nichts gefunden",
+        "heading": {
+          "Post": "Beiträge",
+          "User": "Benutzer",
```


aren't users called `Nutzer` elsewhere in the network? not that it makes a huge difference, just to be consistent :wink:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-08T15:39:51Z" title="Wednesday, January 8th 2020, 4:39:51 pm +01:00">Jan 8, 2020</time>_

---


Unfortunately `Nutzer` and `Benutzer` is used, but the last the most.
And as I have understand `Benutzer` is the preferred form.
Is that right @datenbrei @mattwr18 @roschaefer ?